composer.json

```diff
@@ -1,17 +1,72 @@
 {
-  "name": "alleyinteractive/elasticsearch-extensions",
-  "description": "Elasticsearch Extensions",
-  "type": "wordpress-plugin",
   "authors": [
     {
       "name": "Alley",
-      "email": "noreply@alley.co"
+      "email": "info@alley.co"
     }
   ],
+  "config": {
+    "allow-plugins": {
+      "composer/installers": true,
+      "dealerdirect/phpcodesniffer-composer-installer": true
+    },
+    "sort-packages": true
+  },
+  "description": "Elasticsearch Extensions",
+  "extra": {
+    "installer-paths": {
+      "../../mu-plugins": [
+        "automattic/vip-go-mu-plugins"
+      ],
+      "../{$name}": [
+        "type:wordpress-plugin"
+      ]
+    }
+  },
   "license": "GPL-3.0-or-later",
+  "name": "alleyinteractive/elasticsearch-extensions",
+  "repositories": [
+    {
+      "type": "composer",
+      "url": "https://wpackagist.org",
+      "only": [
+        "wpackagist-plugin/*"
+      ]
+    },
+    {
+      "type": "package",
+      "package": {
+        "name": "automattic/vip-go-mu-plugins",
+        "version": "dev-master",
+        "type": "wordpress-plugin",
+        "source": {
+          "url": "https://github.com/automattic/vip-go-mu-plugins-built",
+          "type": "git",
+          "reference": "master"
+        }
+      }
+    },
+    {
+      "type": "package",
+      "package": {
+        "name": "alleyinteractive/searchpress",
+        "version": "0.4.1",
+        "type": "wordpress-plugin",
+        "source": {
+          "url": "https://github.com/alleyinteractive/searchpress",
+          "type": "git",
+          "reference": "v0.4.1"
+        }
+      }
+    }
+  ],
   "require-dev": {
     "alleyinteractive/alley-coding-standards": "^0.3.0",
+    "alleyinteractive/searchpress": "^0.4.1",
+    "automattic/vip-go-mu-plugins": "dev-master",
+    "composer/installers": "^1.0",
     "phpunit/phpunit": "7.5.*",
```

Comment:
Any risk in letting the polyfills package handle the PHPUnit version?

Reply:
Oh, hey, that's a thought - @renatonascalves is handling tests so I'll seek his advice here. I'm good with it if it doesn't mess anything up for how we have tests set up.

Reply:
No risk. I can update it as part of the test bootstrap.

Reply:
Actually, I remember why I kept this. @kevinfodness would like to keep the composer.lock file and that won't work if we remove it.

Reply:
One of the suggestions outlined here: https://make.wordpress.org/core/2021/09/27/changes-to-the-wordpress-core-php-test-suite/ is precisely removing the composer.lock file from source control so that whatever PHP version is installing the package.

Reply:
That's a good argument in open source plugins like this one that need to test against multiple versions of PHP (in contrast to our site repos where we can target a specific version of PHP and we want to lock down the exact version of dependencies we are using, including plugins). I'd be fine with us removing the lockfile in a separate PR as part of building out the tests. Thanks for doing the research here.

Reply:
Cool! Just a quick reminder that while doing that for SearchPress, you suggested to keep it.

https://github.com/alleyinteractive/searchpress/pull/158#discussion_r920178780